build.gradle.kts

```diff
@@ -8,10 +8,10 @@ import kotlinx.validation.ExperimentalBCVApi
 
 plugins {
     id("kotlinx-io-publish") apply false
+    id("kotlinx-io-dokka")
```

Comment:
Previously we haven't applied Dokka's convention plugin to root, and so it was not configured there